### DISCUSSION TITLE
Ability to see all members of a Family (including step-kids)

### DISCUSSION THREAD

**Discussion by pguedesgit** [PeWu/topola-viewer#173] [Q&A]
First of all, amazing project!
Now to the question, my implementation takes full advantage of the Hourglass and All Relatives views but was wondering if anyone knows how to customize the render to show all the family members instead of only focused on one member.
I have a concrete example where a child from another marriage not on the tree isn't showing unless the parent of that child is focused.

Thanks in advance!

**Comment by PeWu** [comment]
Can you provide more details how to reproduce the issue? What is the family structure exactly that you are thinking of? Maybe you could create a very small GEDCOM file that would show the issue?
